sphinxext/tabs.py

```diff
@@ -24,18 +24,18 @@
 .. raw:: html
 
    <div class="tabs">
-     <ul class="tab-strip tab-strip--singleton" role="tablist">
-       {{ for tab in tabs sortLanguages }}
+     <ul class="tab-strip tab-strip--singleton" role="tablist" PREFERENCE>
+       {{ for tab in tabs FILTER }}
```

Comment:
Should we use some kind of notation with special characters or something to make it less like plain english? For example, $preference

Reply:
I would go with e.g. `%PREFERENCE%`, yeah

Reply:
Done